README.md

```diff
@@ -41,31 +41,66 @@ Note, if you are using the docker MySQL database, ensure that in `src/main/resou
 ## Development
 ### Pre-requisites
 #### Node/Yarn
-The frontend is built using Vue.js. To run the frontend, you will need to have `yarn`. After installing `Node`, run the following to enable `yarn` globally:
+The frontend is built using Vue.js. To run the frontend, you will need to have `yarn`.
+After installing `Node`, run the following to enable `yarn` globally:
 ```bash
 corepack enable
 ```
 (`sudo` may be required)
 
-The frontend can be run in development mode by running the following in the `src/main/resources/frontend` directory:
-```bash
-yarn dev
-```
+#### Backend
+The backend of the app can be run with either `maven`, or by running your own MySQL server.
+
+##### Maven
+Go fish🐟 These instructions are not included in this file.
+
+#### Database
+
+You can run the database inside a Docker container, or locally with your own MySQL server.
 
-#### Docker
-To run the database locally, you will need to have `docker` installed. Run the following in the root of the project to start the db container:
+##### Docker
+Run the following in the root of the project to start the db container:
 ```bash
 docker compose up db -d
 ```
 
-#### Maven
-A local installation of maven is required to run the backend.
+##### MySQL
+Go fish🐟 These instructions are not included in this file.
 
+### Getting Started
+NOTE: These instructions will help you set up this project in Intelli-J.
+If there are any holes or gaps in the instructions, please submit a new pull request
+to share the learned knowledge with the people behind you.
+1. Clone the repo onto your local machine.
+2. Ensure that you are using the latest version of JVM. The system currently requires version 21+.
+3. Use `yarn` to install all dependencies. This only requires running `yarn` in the root directory.
```

Comment:
It'd probably be worth noting that in this case, `root` is defined as `src/main/resources/frontend`